src/padplus-manager/padplus-manager.ts

```diff
@@ -557,6 +557,9 @@ export class PadplusManager extends EventEmitter {
           if (roomRawData) {
             const _data = JSON.parse(roomRawData)
             if (!isRoomId(_data.UserName)) {
+              if (_data.Type7) {
+                log.warn(`This id is not the wxid, please use the wxid to load Contact, otherwise it will make some mistake that unknow.`)
```

Comment:
```
`This id ${id} is not wxid, using weixin instead of wxid will potentially cause system failure. To make sure everything works as excepted, please use the wxid to load Contact.`
```